torch/utils/benchmark/utils/compile.py

```diff
@@ -0,0 +1,175 @@
+import torch
+
+__all__ = ["bench_all", "benchmark_compile"]
+
+import torch._dynamo
+from torch.utils.benchmark import Timer
+
+from typing import Optional, List, Callable, Union, Any, cast
+
+try:
+    from tabulate import tabulate
+    HAS_TABULATE = True
+except ImportError:
+    HAS_TABULATE = False
```

Comment:
nit: Let's just require tabulate to be installed for simplicity, it's not a big deal

Reply:
Alright I'll just add it to requirements.txt

Reply:
You don't have to add it to requirements.txt, the idea is just that to use this file, you'll need tabulate

Reply:
Ah gotcha so still skip test in CI